script.js

```diff
@@ -1546,6 +1546,12 @@ let projectData = [
   {
     projectName: "product Counter",
     projectImage: "assets/Images/product_counter.png",
+    projectUrl: "public/product_counter.html",
+  },
+  {
+    projectName: "Hand Cricket",
+    projectImage: "assets/GIFs/handcricket.gif",
+    projectUrl: "public/handcricket.html",
```

Comment:
Fix this, Two Url are added.
Due to this last one is working, I,e on clicking product counter open.FIx it